homeworks/artur.kobyliatsky_Allocene/GamesProject/style/about.css

```diff
@@ -0,0 +1,249 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+
+/*header section*/
+
+.header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  height: 70px;
+}
+
+.header::before {
+  content: " ";
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: inherit;
+  width: 100%;
+  background-color: #EF4934;
+}
+
+.header__box {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+.header__box-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  z-index: 2;
+}
+
+.header__logo-section {
+  display: flex;
+  align-items: center;
+  z-index: 3;
+}
+
+.header__logo-section img {
+  cursor: pointer;
+}
+
+.header__menu-section {
+  display: flex;
+  align-items: center;
+  z-index: 2;
+}
+
+.header__menu-section a {
+  text-decoration: none;
+  color: #FFF;
+}
+
+.header__menu-section a:active {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.header__menu-section a:hover {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.menu__list {
+  display: flex;
+  z-index: 2;
+}
+
+.menu__list li {
+  list-style: none;
+  margin: 0 0 0 20px;
+}
+
+.menu__list-link {
+  font-size: 16px;
+  text-decoration: none;
+}
+
+
+/*main section*/
+
+main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.contain {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 140px;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 68px;
+  justify-content: center;
+  margin-bottom: 150px;
+}
+
+.img {
+  align-items: start;
+}
+
+.text {
+  width: 575px;
+  height: 478px;
+}
+
+.text h1 {
```

Comment:
Better avoid styling by tag name. Better add a proper className

Reply:
Fixed

Reply:
And now, you can see, what happened on the Games page. The tag selector has more weight than the styles inherited from the cascade `.content .text`. 
So, the rule of thumb is: one element - one class. You can apply here same class `.text__heading` as on the About page

Reply:
Done